layouts/shortcodes/influx-cli-global-flags.md

```diff
@@ -0,0 +1,9 @@
+## Global Flags
+
+| Flag            | Description                                                                                 | Input type |
+|:----------------|:--------------------------------------------------------------------------------------------|:----------:|
+| `-h`, `--help`  | Help for the influx command                                                                 |            |
```

Comment:
While `--help` is technically global, it's description depends on its context. You either need to remove this from the global flags shortcode, or remove the command specific help flags from each command. Probably easier to remove this one.